flink-table-store-connector/src/test/java/org/apache/flink/table/store/connector/sink/CommittableSerializerTest.java

```diff
@@ -18,21 +18,74 @@
 
 package org.apache.flink.table.store.connector.sink;
 
+import org.apache.flink.core.io.SimpleVersionedSerializer;
+import org.apache.flink.table.store.file.mergetree.Increment;
+import org.apache.flink.table.types.logical.IntType;
+import org.apache.flink.table.types.logical.RowType;
+
 import org.junit.jupiter.api.Test;
 
+import java.io.IOException;
+
+import static org.apache.flink.table.store.file.manifest.ManifestCommittableSerializerTest.randomIncrement;
+import static org.apache.flink.table.store.file.mergetree.compact.CompactManagerTest.row;
 import static org.assertj.core.api.Assertions.assertThat;
 
 /** Test for {@link CommittableSerializer}. */
 public class CommittableSerializerTest {
 
+    private final FileCommittableSerializer fileSerializer =
+            new FileCommittableSerializer(
+                    RowType.of(new IntType()),
+                    RowType.of(new IntType()),
+                    RowType.of(new IntType()));
+
+    private final CommittableSerializer serializer =
+            new CommittableSerializer(fileSerializer, (SimpleVersionedSerializer) fileSerializer);
```

Comment:
Nit: I'm not sure whether it's a good practice to pass `fileSerializer` as `logCommittableSerializer` for convenience.  These two serializer types should be orthgoal.

Reply:
Let's use StringCommittableSerializer